.gitignore

```diff
@@ -1,33 +1,34 @@
 /.artifacts/
-/_output/
+/bin/
 /brew
+/build/
+/cmd/podman/varlink/iopodman.go
+/cmd/podman/varlink/ioprojectatomicpodman.go
 /conmon/
+contrib/spec/podman.spec
+*.coverprofile
 /docs/*.[158]
 /docs/*.[158].gz
-/docs/remote
 /docs/build/
+/docs/remote
+.gopathok
+.idea*
+.nfs*
```

Comment:
Let's put changes like this in dedicated PRs. Makes review much harder if it's not clear what has been added or removed in the first place :innocent:

Reply:
Fixed